protobuf/v1/replica.proto

```diff
@@ -119,6 +119,7 @@ message ReplicaSnapshot {
   uint64 replica_size = 7; // amount of bytes referenced by replica
   string entity_id = 8; // uuid of the entity for which snapshot is taken.
   string txn_id = 9; // unique transaction id for snapshot.
+  bool valid_snapshot = 10; // true: valid, false: invalid (missing snapshotdescriptor field)
```

Comment:
so just to confirm, if invalid, then some descriptor fields will be invalid?
If so, could we highlight which fields come from snapshotdescriptor?

Reply:
> so just to confirm, if invalid, then some descriptor fields will be invalid? If so, could we highlight which fields come from snapshotdescriptor?

Updated the field details which comes part of descriptor and leads to make valid_snapshot as false.